commit 6b1b60f41eef3ba7b188fd72f1d6de478aafd93c
Author: Don Mullis <dwm@meer.net>
Date:   Fri Dec 8 02:39:53 2006 -0800

    [PATCH] fault-injection: defaults likely to please a new user
    
    Assign defaults most likely to please a new user:
     1) generate some logging output
        (verbose=2)
     2) avoid injecting failures likely to lock up UI
        (ignore_gfp_wait=1, ignore_gfp_highmem=1)
    
    Signed-off-by: Don Mullis <dwm@meer.net>
    Cc: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@osdl.org>
    Signed-off-by: Linus Torvalds <torvalds@osdl.org>

diff --git a/include/linux/fault-inject.h b/include/linux/fault-inject.h
index c77067916b7e..32368c4f0326 100644
--- a/include/linux/fault-inject.h
+++ b/include/linux/fault-inject.h
@@ -52,6 +52,7 @@ struct fault_attr {
 		.times = ATOMIC_INIT(1),			\
 		.require_end = ULONG_MAX,			\
 		.stacktrace_depth = 32,				\
+		.verbose = 2,					\
 	}
 
 #define DECLARE_FAULT_ATTR(name) struct fault_attr name = FAULT_ATTR_INITIALIZER